src/java/staticstructures/StaticStack.java

```diff
@@ -0,0 +1,72 @@
+package staticstructures;
+
+import definitions.IStack;
+
+public class StaticStack implements IStack {
+
+    private final int[] elements;
+    private int lastElementIndex;
+
+
+    @Override
+    public void push(int element) {
+        if (isFull()) {
+            System.out.println("Stack is full");
```

Comment:
This can be misleading. If stack is full, throw exception so the user of the method knows that element is not added. Writing the log is not enough